tests/integration/tables/yara.cpp

```diff
@@ -22,26 +22,24 @@ class yara : public testing::Test {
 };
 
 TEST_F(yara, test_sanity) {
-  // 1. Query data
-  auto const data = execute_query("select * from yara where path = ''");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See helper.h for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidationMap row_map = {
-  //      {"path", NormalType}
-  //      {"matches", NormalType}
-  //      {"count", IntType}
-  //      {"sig_group", NormalType}
-  //      {"sigfile", NormalType}
-  //      {"strings", NormalType}
-  //      {"tags", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+  // Query data from yara table
+#if !defined(WINDOWS)
+  auto const data = execute_query(
+      "select * from yara where path like '%' and sigrules = "
```

Comment:
I feel naive. What should this return? I'm a bit bothered by the `0ul` designation

Reply:
This will scan for the `sigrules` on files in the current directory. The check for the `data.size() >= 0` is to avoid the failure if the current directory has no files.

Reply:
If the directory has no files, then the test isn't testing anything and is invalid. 

I'd suggest making a temp file, and feeding that to the yara scan. It will remove the ambiguity.